routes/api/userRoutes.js

```diff
@@ -18,19 +23,26 @@ router.post('/login', async (req, res) => {
       res
         .status(400)
         .json({ message: 'Incorrect  password, please try again' });
+        console.log("Successfull Babe");
       return;
     }
 
     req.session.save(() => {
       req.session.user_id = userData.id;
       req.session.logged_in = true;
+      req.session.cookie
       
       res.json({ user: userData, message: 'You are now logged in!' });
     });
 
   } catch (err) {
     res.status(400).json(err);
   }
+  // Get the authentication status from the session or any other mechanism
+  const isAuthenticated = req.session.isAuthenticated || false;
```

Comment:
Why we are going to use `isAuthenticated`? where it comes from? where we change it? we already have req.session.logged_in, are these two have the same function?